src/components/shared/MarkerContextMenu.js

```diff
@@ -258,61 +258,63 @@ class MarkerContextMenuImpl extends PureComponent<Props> {
     return (
       <ContextMenu
         id="MarkerContextMenu"
+        className="markerContextMenu"
         onShow={this._onShow}
         onHide={this._onHide}
       >
+        <MenuItem
```

Comment:
This is mostly moving things around with some label changes after discussions with Florian.